policyengine_canada/variables/household/person/children.py

```diff
@@ -1,10 +1,10 @@
 from policyengine_canada.model_api import *
 
 
-class child(Variable):
+class children(Variable):
     value_type = int
-    entity = Person
+    entity = Household
     label = "Children"
     unit = CAD
-    documentation = "Number of dependent children under the age of 18"
+    documentation = "Number of dependent children under the age of 19"
```

Comment:
Is under-19 a general rule across programs? Does CCB have the same rules?

If the climate action incentive defines this specifically, it should be a policy parameter tied to the program.

Reply:
please confirm my q

Reply:
Got it, in that case looks like you're on the right track with `climate_action_incentive_dependent_children`